package.json

```diff
@@ -22,7 +22,8 @@
     "onView:targetTree",
     "onCommand:browser-preview.openPreview",
     "onDebugInitialConfigurations",
-    "onDebug"
+    "onDebug",
+    "onFileSystem:vsls"
```

Comment:
This simply activates the extension whenever a guest joins into a Live Share session. This is necessary since the host might open a preview window at any time, and we'd like to automatically open that up on the guest side as well.